### DISCUSSION TITLE
Decision tree

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/MLAlgorithms#24] [Q&A] [answered]
sklearn decision tree classifier uses which algorithm by default?

**Comment by Mazin7d4** [comment]
The DecisionTreeClassifier in sklearn uses the CART (Classification and Regression Trees) algorithm by default. This algorithm creates binary splits of the data to build the tree.

**Comment by Haarika78** [comment, marked as answer]
The DecisionTreeClassifier in sklearn uses the CART (Classification and Regression Trees) algorithm by default. This algorithm algorithm helps in building the trees
